app/views/pages/security.html.erb

```diff
@@ -0,0 +1,211 @@
+      download corruption protection. It is not intended to provide any sort of
+      security regarding tampering. Please use GPG signing for that.
+    </p>
+
+    Also, see <a href="https://gist.github.com/bf4/4223e83e8becacfb2a8e">this gist</a>
+
+    <h2>Reporting RubyGems.org Website Problems</h2>
+    <p>
+      To report a bug or to suggest an enhancement,
+      send an e-mail to <webmaster@rubygems.org>.
+    </p>
+
+    <p>
+      If you're reporting a problem (a broken link, typo,
+      formatting glitch, etc),  please include the URL of the affected page.
+    </p>
+
+    <p>
+      Note that this address is only for problems with the main rubygems.org
+      website.
+    </p>
+
+    <p>
+      For problems with RubyGems.org please use the
+      <a href="https://github.com/rubygems/rubygems.org">RubyGems.org issue tracker</a>.
+    </p>
+
+    <h2>Disclosure Policy</h2>
+    <p>
+      RubyGems.org has a 5 step disclosure policy.
+    </p>
+
+    <ol>
+      <li>
+        Security report received and is assigned a primary handler. This person will coordinate the fix and release process.
+      </li>
+      <li>
+        Problem is confirmed and, a list of all affected versions is determined. Code is audited to find any potential similar problems.
+      </li>
+      <li>
+        Fixes are prepared for all releases which are still supported. These fixes are not committed to the public repository but rather held locally pending the announcement.
+      </li>
+      <li>
+        A suggested embargo date for this vulnerability is chosen and distros@openwall is notified. This notification will include patches for all versions still under support and a contact address for packagers who need advice back-porting patches to older versions.
+      </li>
+      <li>
+        On the embargo date, the RubyBems security mailing list is sent a copy of the announcement. The changes are pushed to the public repository and new gems released to rubygems. At least 6 hours after the mailing list is notified, a copy of the advisory will be published on
+        the <a href="http://blog.rubygems.org/">RubyGems.org blog</a>.
+      </li>
+    </ol>
+
+    <p>
+      Typically the embargo date will be set 72 hours from the time vendor-sec
+      is first notified, however this may vary depending on the severity of the
+      bug or difficulty in applying a fix.
+    </p>
+
+    <p>
+      This process can take some time, especially when coordination is required
+      with maintainers of other projects. Every effort will be made to handle
+      the bug in as timely a manner as possible, however it’s important that
+      we follow the release process above to ensure that the disclosure is
+      handled in a consistent manner.
+    </p>
+
+    <h2>Receiving Security Updates</h2>
+
+    <p>
+      The best way to receive all the security announcements is to subscribe
+      to the rubygems security mailing list.
```

Comment:
This is confusing... wouldn't the 'rubygems security mailing list' be security@rubygems.org, which isn't publicly joinable... Perhaps you mean a secondary security announce list, or just reusing the existing rubysec-announce@ or ruby-security-ann@ lists...